cumulusci/tasks/release_notes/parser.py

```diff
@@ -186,7 +186,7 @@ class GithubIssuesParser(IssuesParser):
 
     def __new__(cls, release_notes_generator, title, issue_regex=None):
         if not release_notes_generator.has_issues:
-            logging.getLogger(__file__).warn(
+            logging.getLogger(__file__).warning(
```

Comment:
As of Python 3.3, logger.warn has been deprecated and logger.warning must be used. Even though logger.warn is still backwards compatible, you should not use it.